include/simpleble/Types.h

```diff
@@ -5,20 +5,20 @@
 
 namespace SimpleBLE {
 
-typedef std::string BluetoothAddress;
+using BluetoothAddress = std::string;
```

Comment:
Nice, I didn't know this.

Reply:
`typedef` is equivalent to `using` for introducing a type alias, _except_ `using` also works with templates, whereas `typedef` does not, so I think it's better to just stick with `using` everywhere.